packages/ai/core/generate-image/generate-image-result.ts

```diff
@@ -19,6 +20,11 @@ The images that were generated.
 Warnings for the call, e.g. unsupported settings.
      */
   readonly warnings: Array<ImageGenerationWarning>;
+
+  /**
+Response metadata from the provider. There may be multiple responses if we made multiple calls to the model.
+   */
+  readonly responses: Array<ImageModelResponseMetadata | undefined>;
```

Comment:
We could make this mandatory -- current image models all return their response headers. This follows what I see in `language-model-v1.ts` where `response` and `rawResponse` are optional.

Reply:
lets' make it mandatory - it is only optional in language models for backwards compat